### PR TITLE
Fixing Typos

### PR DESCRIPTION
**Pull request by gep13**
- Made use of the ReSpeller Extension for ReSharper

### CODE REVIEW

**Comment by DevWouter** [comment]
There are a few cases where `_field` is replaced by `this._field` since the rest of the source doesn't use this you might want restore this to the original behaviour.

**Comment by gep13** [comment]
@DevWouter  Ah, good catch!  That is ReSharper getting ahead of itself, I will correct.

**Comment by gep13** [comment]
I have fixed up the issue pointed out by @DevWouter

**Comment by DevWouter** [comment]
Looks good now. :+1:

File: src/Cake.Common/Tools/Chocolatey/ApiKey/ChocolateyApiKeySettings.cs [hunk 1]
```diff
@@ -55,6 +55,6 @@ public sealed class ChocolateyApiKeySettings : ToolSettings
         /// Gets or sets a value indicating whether to run in allow unofficial mode.
         /// </summary>
         /// <value>The allow unofficial flag</value>
-        public bool AllowUnoffical { get; set; }
+        public bool AllowUnofficial { get; set; }
```
Comment (devlead) [inline src/Cake.Common/Tools/Chocolatey/ApiKey/ChocolateyApiKeySettings.cs]: This is a breaking change as it's an public API, though new so perhaps now's the time. Don't think many have taken an dependency on this yet, but for those that have their code will break.

File: src/Cake.Common/Tools/Chocolatey/Pack/ChocolateyPackSettings.cs [hunk 2]
```diff
@@ -186,6 +186,6 @@ public sealed class ChocolateyPackSettings : ToolSettings
         /// Gets or sets a value indicating whether to run in allow unofficial mode.
         /// </summary>
         /// <value>The allow unofficial flag</value>
-        public bool AllowUnoffical { get; set; }
+        public bool AllowUnofficial { get; set; }
```
Comment (devlead) [inline src/Cake.Common/Tools/Chocolatey/Pack/ChocolateyPackSettings.cs]: Same here.

**Comment by devlead** [comment]
@gep13 think it looks good, there's breaking changes which we really try to avoid. But in this case makes sense to do an exception,

Please get branch up to date and I'll look gain.

**Comment by gep13** [comment]
@devlead rebased and pushed.

That is a very good point about the potential breaking change.  Although in this case, the Chocolatey Alias is slated for release in 0.5.6, so not currently available outwith MyGet Sources, so I "think" in this instance, we are ok.  Right?

**Comment by devlead** [comment]
@gep13 yes in this case it's "ok", it's just get a bad gut feeling when seeing rename of public members :wink:

**Pull request merged**